src/main/java/edu/cornell/kfs/module/purap/service/impl/CuElectronicInvoiceHelperServiceImpl.java

```diff
@@ -1430,9 +1430,9 @@ protected StringBuffer buildLoadSummary(ElectronicInvoiceLoad eInvoiceLoad) {
         finalText.append("======================================\n");
         finalText.append("               TOTALS\n");
         finalText.append("======================================\n\n");
-        finalText.append(ACCEPT).append("           : ").append(loadCounts.get(ACCEPT)!=null?loadCounts.get(ACCEPT):0).append("\n");
-        finalText.append(REJECT).append("           : ").append(loadCounts.get(REJECT)!=null?loadCounts.get(REJECT):0).append("\n");
-        finalText.append(EXTRACT_FAILURES).append(" : ").append(loadCounts.get(EXTRACT_FAILURES)!=null?loadCounts.get(EXTRACT_FAILURES):0).append("\n");
+        finalText.append(ACCEPT).append("           : ").append((Integer)(loadCounts.get(ACCEPT)!=null?loadCounts.get(ACCEPT):0)).append("\n");
+        finalText.append(REJECT).append("           : ").append((Integer)(loadCounts.get(REJECT)!=null?loadCounts.get(REJECT):0)).append("\n");
+        finalText.append(EXTRACT_FAILURES).append(" : ").append((Integer)(loadCounts.get(EXTRACT_FAILURES)!=null?loadCounts.get(EXTRACT_FAILURES):0)).append("\n");
```

Comment:
Good catch, that's a Java 8 vs. Java 7 thing. The POC code I pulled in and modified for DynamoDB requires Java 8, and without this cast, this code won't compile, I think it had to do with multiple possible return types being valid - Object vs. Integer.